dev/publicDev/photos/tpl.jade

```diff
@@ -7,5 +7,8 @@
           img(ng-src="{{photo}}")
           p {{photo.updated}}
           p Driver {{photo.ownerNumber}}
-        .message(ng-if="!vm.photos.length")
+        .photo(ng-repeat="photo in vm.emergencyTowing")
+          img(ng-src="{{photo}}")
+          p Property: {{photo.propertyName}}
+        .message(ng-if="!vm.photos.length && !vm.emergencyTowing.length")
```

Comment:
vm.photos it`s photo for one property and vm.emergencyTowing its for all property